### PR TITLE
Add a migration script from MLflow machine learning platform

### PR DESCRIPTION
**Pull request by tmankita**

### CODE REVIEW

**Comment by bmartinn** [comment]
Hi @tmankita ,
What's the interface for using the migration tool ?

**Comment by tmankita** [comment]
> Hi @tmankita,
> What's the interface for using the migration tool?

Hi,
The migration tool is a command-line (interface excluded). You can read more details about migration API in the following [link](https://github.com/tmankita/trains/tree/master/migrant_script).
currently support two ways of migration 
1. From files on the local computer.
2. From DataBase.

File: migrant_script/README.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Supported paths:
+
+- Local Computer : ``file://path_to_mlruns_folder``
+- SQLAlchemy database URI (e.g. ``<dialect>+<driver>://<username>:<password>@<host>:<port>/<database>`` )
+- mlflow tracking UI server address (e.g. ``http://<IP>:<PORT>`` )
+
+Run::
+
+    python trains_immigration.py <path
```
Comment (Ben-Epstein) [inline migrant_script/README.rst:25]: missing `>`

**Review by Ben-Epstein — COMMENTED** [review]
just a missing `>`

**Comment by pievalentin** [comment]
would be great to have it merge, to move my team to clearml!
I can fix the typo in the rst file

**Comment by jkhenning** [comment]
Hi @pievalentin,

> would be great to have it merge, to move my team to clearml!

That's great to hear 🙂 

> I can fix the typo in the rst file

Well, I think this PR is a little out-dated (predates our move from trains to clearml), so there's some more work here, as well as required testing.

If you're willing to contribute here, we'll really appreciate it (and assist as best we can 😄 )

**Comment by bmartinn** [comment]
@Ben-Epstein, @pievalentin  did test this PR lately ? Any idea how far it is from working again ?

**Pull request closed without merging**